content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md

```diff
@@ -41,13 +45,13 @@ To update your development environment:
 
 1. Open a terminal shell on your computer.
 
-2. Update your Rust environment by running the following command:
+1. Update your Rust environment by running the following command:
```

Comment:
```suggestion
2. Update your Rust environment by running the following command:
```

Reply:
I believe this change is because markdown automatically increments numbers? e.g. 

1. One
1. Two
1. Three

![image](https://user-images.githubusercontent.com/75586/217479624-7111b5d8-6da5-4b9e-8c09-6f10003749ca.png)